packages/react-liveness/src/components/FaceLivenessDetector/service/utils/liveness.ts

```diff
@@ -274,8 +274,6 @@ export function drawStaticOval(
   const ovalDetails = getStaticLivenessOvalDetails({
     width: width!,
     height: height!,
-    centerXSeed: Math.round(Math.random() * 10) / 10, // random values between 0 and 1
-    centerYSeed: Math.round(Math.random() * 10) / 10, // random values between 0 and 1
```

Comment:
Do we need to do any cleanup in `getStaticLivenessOvalDetails`?

Reply:
Nope, this is just reverting changes that were made just for the DA's to do testing